pom.xml

```diff
@@ -92,6 +92,7 @@
     <module>mllib-local</module>
     <module>tools</module>
     <module>streaming</module>
+    <module>sql/parser</module>
```

Comment:
Introduced new module and moved `.g4` files to the new module where `antlr4-runtime` is shaded. sql-api and other modules that use `antlr4` will now depend on the new module. This avoids https://github.com/apache/maven-shade-plugin/issues/345.